acceptance/scripts/generic/testrun.sh

```diff
@@ -1,10 +1,12 @@
 #!/bin/bash
 set -x
 
-export GEM_SOURCE=http://rubygems.delivery.puppetlabs.net
+# get to the correct directory
+while [ ! -d acceptance ] ; do cd ..; done
```

Comment:
Seems like this could get into an infinite loop depending upon where you are when it starts?  I generally don't like scripts changing my current working directory while they execute.  My preference would be to remove this.